### ISSUE TITLE
`sci-zurich` - add new subjects

### ISSUE THREAD

**Issue by valosekj** [neuropoly/data-management#224]
Within `jv/add_new_subjects` branch and `e7a6c4012badef23df4428ecee895467da434cc9` commit, I added 17 new subjects (sub-zh82 to sub-zh98) to `sci-zurich` dataset.

I also updated the `README`, the `participants.tsv` file, and I added the script I used under `/code`.

**Comment by mguaypaq** [comment]
This looks fine with respect to git-annex and bids-validator. Merged into master!

**Comment by naga-karthik** [comment]
Reopening this as I found an issue in the naming of the derivative files for the new subjects. It seems that `_` is missing after subject and session names. This is the case for all new subjects (I have shown only 2 of them here). Note that the corresponding image files are correctly named, the issue is only with the derivative files

<img width="552" alt="Screen Shot 2023-03-29 at 1 11 41 PM" src="https://user-images.githubusercontent.com/53445351/228461802-26485631-7ad9-4fe5-83c9-4cf64c527005.png">

@valosekj I noticed a `code/` folder in the root directory. Should the correction be made in the `add_new_subjects_to_sci-zurich.py` file?

**Comment by valosekj** [comment]
Good catch, Naga!

> I noticed a code/ folder in the root directory. Should the correction be made in the add_new_subjects_to_sci-zurich.py file?

Yes, this would make sense. But the source data seems to be deleted. 
Since this fix is only about adding `_`, I fixed the derivatives filenames for 17 newly added subjects by the following command:

```console
$ cd derivatives/labels
$ for file in */*/*/sub-zh[0-9][0-9]ses*;do mv ${file} ${file/ses-01acq/_ses01_acq};done
```

I pushed 51 fixed files (17 subjects; each has nii, json and roi files) in a new branch, `jv/add_missing_underscore_to_derivatives_files`. @mguaypaq, could you please check it and merge it?

**Comment by naga-karthik** [comment]
Thanks for the quick fix, @valosekj! 

Just a minor correction. I think there should a hyphen `-` between `ses` and `01`. Something like,

```bash
$ cd derivatives/labels
$ for file in */*/*/sub-zh[0-9][0-9]ses*;do mv ${file} ${file/ses-01acq/_ses-01_acq};done
```

This should rename all files correctly!

**Comment by valosekj** [comment]
> Just a minor correction. I think there should a hyphen `-` between `ses` and `01`. Something like,

Great catch again! Indeed, I forgot `-` :-(

I fixed filenames in `cb95905719497e57513dc09e4f3ce68081254f60` commit.

**Comment by mguaypaq** [comment]
Merged into master, thanks for the fix!

As a side note, I guess this kind of filename issue might be caught by running `bids-validator derivatives/labels/` in the future, if we can figure out the right way to do it.
